src/components/visualizations/chart/highcharts/plugins/zeroAlignYAxis.ts

```diff
@@ -0,0 +1,287 @@
+// tslint:disable-line
+/**
+ * Highcharts extension that zero-aligns the scales of primary and secondary y-axes.
+ * Author:  Christos Koumenides
+ * Page:    https://www.highcharts.com/products/plugin-registry/single/42/Zero-align%20y-axes
+ * Github:  https://github.com/chriskmnds/highcharts-zero-align-y-axes
+ *
+ * Modified by binh.nguyen@gooddata.com to support min/max configuration
+ */
+
+import get = require('lodash/get');
```

Comment:
a lot of functions in this file just manipulate basic numbers, so can be easylly tested

Reply:
@ivan-nejezchleb , what do you mean here? can you explain more?

Reply:
this plugin contains a lot of functions. Write unit tests for them, eg. handleOutOfRange mentions some example in comment, but it should be described in unit test

Reply:
will be resolved in separate PR